server/re/src/mailMS.cpp

```diff
@@ -46,6 +56,24 @@
  * \sa none
  **/
 int msiSendMail( msParam_t* xtoAddr, msParam_t* xsubjectLine, msParam_t* xbody, ruleExecInfo_t* ) {
+    const auto enabled = [] {
+        try {
+            return irods::get_advanced_setting<bool>("enable_msiSendMail");
```

Comment:
we had originally named this `enable_deprecated_msiSendMail`.      in case that wasn't an intentional choice to shorten it.

Reply:
Will change.

Reply:
Done